android/services/mongodb-remote-perftests/src/androidTest/java/com/mongodb/stitch/android/services/mongodb/performance/SyncL2ROnlyPerformanceTestDefinitions.kt

```diff
@@ -176,17 +146,15 @@ class SyncL2ROnlyPerformanceTestDefinitions {
             pctOfDocsWithChangeEvents: Double
         ) {
             val testName = "testL2R_SyncPass_${pctOfDocsWithChangeEvents}DocsChanged"
-            Log.d(TAG, testName)
 
             // Local variable for the number of docs updated in the test
             // This should change for each iteration of the test.
-            var numberOfChangedDocs: Int? = null
+            var numberOfChangedDocs = -1
```

Comment:
[nit] assuming that this is to deal with nullability, is this fail-able? If so, it should remain as such